doc/async_worker.md

```diff
@@ -140,16 +140,15 @@ operations ends. The given function is called from the main event loop thread.
 identifier for the kind of resource that is being provided for diagnostic
 information exposed by the async_hooks API.
 
-Returns an AsyncWork instance which can later be queued for execution by calling
+Returns a `Napi::AsyncWork` instance which can later be queued for execution by calling
 `Queue`.
```

Comment:
nit: &#96;Queue&#96; should be fully namespaced.

Reply:
Done